src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -511,10 +511,11 @@ function showVersionWarningBanner(data) {
     localStorage.getItem("pst_banner_pref") || "{}",
   )[version];
   if (dismiss_date_str != null) {
-    let dismiss_date = new Date(dismiss_date_str);
-    let now = new Date();
-    let days_passed = now.getDate() - dismiss_date.getDate();
-    let timeout_in_days = 14;
+    const dismiss_date = new Date(dismiss_date_str);
+    const now = new Date();
+    const milisecond_in_a_day = 24 * 60 * 60 * 100;
+    const days_passed = (now - dismiss_date) / milisecond_in_a_day;
```

Comment:
```suggestion
    const millisecond_in_a_day = 24 * 60 * 60 * 1000;
    const days_passed = (now - dismiss_date) / millisecond_in_a_day;
```

missed a zero I think.  Also I corrected the varname spelling.

Reply:
Ghaaaa... we'll get there.